wiki/Help_centre/Upgrading_to_lazer/it.md

```diff
@@ -0,0 +1,284 @@
+
+#### le vecchie skin funzioneranno nella selezione delle mappe e nella schermata del risultato?
+
+Faremo del nostro meglio per riportare queste funzionalità 
+
+#### posso usare il cursore della skin pure sul menu'?
+
+porteremo molto probabilmente questa novita' date la richiesta elevata.
+
+### Prestazioni
+
+#### perche' non posso avere gli fps illimitati?
+
+Al di sopra di una certa soglia, non c'è motivo di utilizzare frame rate più elevati. Lazer impiega diverse nuove tecnologie per garantire la latenza più bassa possibile senza richiedere frame rate elevati. Questo aspetto continuerà a migliorare in futuro, poiché abbiamo ancora ulteriori miglioramenti da implementare.
+
+Lazer esegue il polling per l'ingresso a 1000 Hz indipendentemente dal limitatore di FPS, motivo per cui anche l'impostazione massima del limitatore limiterà a 1000 FPS.
+
+Se siete curiosi di sapere come questo influisce sulla latenza in ingresso e di verificare la vostra percezione, eseguite il "certificatore di latenza" integrato in fondo alle impostazioni.
+
+#### Se il polling dell'input viene eseguito solo a 1000 Hz, che ne è del mio mouse da gioco a 8000 Hz?
+
+Il sistema operativo continuerà a eseguire il polling alla frequenza più alta possibile, anche se i vantaggi sono dimostrati essere irrilevanti. Il polling a velocità così elevate può comportare richiesta di potenza aggiuntiva imprevista e si consiglia di limitare le proprie periferiche a 1000 Hz per garantire una esperienza stabile.
+
+#### Lazer non funziona in modo stabile. Cosa succede?
+
+Sebbene sulla maggior parte dell'hardware moderno notiamo che lazer supera stable, però, ci sono sempre casi dove utenti non hanno una configurazione hardware performante. Nella nostra roadmap a breve termine, stiamo cercando di supportare DirectX (alias "modalità compatibile" su stable) e Vulkan, che hanno entrambi un supporto driver migliore di OpenGL su tutto l'hardware. Una volta implementato, le prestazioni su hardware come i chipset integrati Intel miglioreranno notevolmente.
+
+### Fornire feedback
+
+#### Manca una funzione da cui dipendo! / Qualcosa è cambiato e non mi piace. / Ho trovato un bug, qual è il modo migliore per segnalarlo?
+
+È molto probabile che ne siamo già a conoscenza e che lo stiamo monitorando per una futura implementazione! Cerca nell'[issue tracker](https://github.com/ppy/osu/issues) e nella [pagina delle discussioni](https://github.com/ppy/osu/discussions). Se non si riesce a trovare nessuna discussione corrispondente, sentirsi liberi di [aprire una discussione](https://github.com/ppy/osu/discussions/new).
+
+Si noti che stiamo già monitorando più di 1.000 problemi di varia priorità e che potrebbe volerci del tempo per risolvere problemi che riguardano solo un piccolo numero di utenti.
+
+### Altro
+
+#### Perché ci vuole così tanto tempo per diventare la versione "principale"?
```

Comment:
i think that we can't make the subject implicit in this sentence it would be better to rephrase it to something like "Perché lazer ci sta mettendo così tanto tempo per diventare la versione "principale"?"